apps/examples/react-ssr-example/src/client/index.tsx

```diff
@@ -1,8 +1,8 @@
 import { Router } from "@pixeloven-react/routing";
-import { App } from "@shared/components";
+import App from "@shared/components/App";
```

Comment:
Does this mean we need to update all our project imports to be direct instead of grouped?

Reply:
Not sure if we need to do this for SSR. I was sorta hoping to find another way to do this but these changes get this example to 0 circular deps.